build.sbt

```diff
@@ -34,50 +34,24 @@ lazy val root = (project in file("."))
   .settings(noPublishSettings)
   .settings(name := "Trace4Cats")
   .aggregate(
-    base,
-    `base-io`,
-    `base-laws`,
-    core,
+    `fp-utils`,
+    `fp-utils-io`,
+    `fp-utils-laws`,
     `dynamic-sampling`,
     `dynamic-sampling-config`,
-    `exporter-common`,
-    `exporter-stream`,
     filtering,
     fs2,
-    inject,
     `inject-io`,
     kernel,
-    `log-exporter`,
     meta,
-    model,
-    `model-testkit`,
-    `rate-sampling`,
     `tail-sampling`,
     testkit
   )
 
-lazy val model =
-  (project in file("modules/model"))
-    .settings(publishSettings)
-    .settings(
-      name := "trace4cats-model",
-      libraryDependencies ++= Seq(
-        Dependencies.catsEffectStd,
-        Dependencies.commonsCodec,
-        // Dependencies.kittens, // TODO re-add once compatible with Scala 3
-        Dependencies.caseInsensitive
-      )
-    )
-
-lazy val `model-testkit` = (project in file("modules/model-testkit"))
```

Comment:
Will this affect other repos, where it may be used?

Reply:
I don't think so, it was a bit of a hack that allowed `kernel` to be tested.